modules/subscription/main.tf

```diff
@@ -35,4 +35,34 @@ resource "azapi_resource" "subscription" {
     }
   })
   response_export_values = ["properties.subscriptionId"]
+  lifecycle {
+    ignore_changes = [
+      body
+    ]
+  }
+}
+
+resource "azapi_update_resource" "subscription_tags" {
+  count = var.subscription_alias_enabled && var.subscription_use_azapi ? 1 : 0
+
+  type        = "Microsoft.Resources/tags@2022-09-01"
+  resource_id = "/subscriptions/${jsondecode(azapi_resource.subscription[0].output).properties.subscriptionId}/providers/Microsoft.Resources/tags/default"
+  body = jsonencode({
+    properties = {
+      tags = var.subscription_tags
```

Comment:
Tags are still specified in the alias resource. Are they still needed there?

Reply:
Could cause a breaking, if I removed the arg no?

Reply:
It is possible I am not understanding how AzAPI works. Does the alias create them and then the PUT is only used if there is an update? What happens to the alias resource if someone updates the tags? Won't it try to update that resource too? Same for subscription name?

Reply:
Creating an alias is a write-once scenario as I have been told, so updates do nothing practically after the first apply, but do cause the terraform to think there's an update causing a bit of noise.

Reply:
Right, got it. I see now. Ok, LGTM then.